tests/responses/test_java.py

```diff
@@ -40,6 +40,24 @@ class TestJavaStatusResponse(BaseResponseTest):
     def build(self) -> JavaStatusResponse:
         return JavaStatusResponse.build(self.RAW)  # type: ignore # dict[str, Unknown] cannot be assigned to TypedDict
 
+    def test_as_json(self, build: JavaStatusResponse):
```

Comment:
This should probably also be tested on `BedrockStatusResponse`.

Reply:
Done [here](https://github.com/py-mine/mcstatus/compare/2a357cf771b6ce1c26476887675a1c15c99cf9ec..a463a49c94c0b2bb9e8cb47e036db772f3c8c477)